src/trans-word-level/property.cpp

```diff
@@ -72,7 +72,7 @@ Function: property
 
 void property(
   const exprt &property_expr,
-  exprt::operandst &prop_handles,
+  std::map<std::size_t, exprt::operandst> &obligations,
```

Comment:
Shouldn't this actually become the return value?

Reply:
Yes, will do.